org.eclipse.xtext.builder.standalone.tests/build.gradle

```diff
@@ -3,6 +3,7 @@ dependencies {
 	compile project(':org.eclipse.xtext.ecore')
 	compile "org.eclipse.xtext:org.eclipse.xtext.testing:$versions.xtext"
 	compile "org.eclipse.jdt:org.eclipse.jdt.core:$versions.jdtCore"
+	compile "org.eclipse.jdt:org.eclipse.jdt.compiler.apt:$versions.jdtCompilerApt"
```

Comment:
It's not necessary to add the dependency here: compile dependencies in Gradle are always transitive.